components/brave_new_tab_ui/brave_new_tab.html

```diff
@@ -20,6 +20,21 @@
   #root {
     height: 100%;
   }
+
+  ::-webkit-scrollbar {
```

Comment:
Can you test using `scrollbar-width: thin` instead
https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width

I'm pretty sure it's shipped recently and `::-webkit-scrollbar` might be deprecated.

Reply:
I actually used the newer properties initially, but discarded them to instead use already-accepted code. I assumed this would have been more ideal, and expedite the approval process :) I will check those again to see if the same (or a sufficiently similar) effect can be achieved.